packages/scss/src/components/_actionIcon.scss

```diff
@@ -10,11 +10,24 @@
 	width: 2rem;
 
 	&:hover {
-		background: _component("actionIcon.hover.background");
-		color: darken(_color("text.default"), 25%);
+		&:not([disabled]) {
```

Comment:
Isn't possible to remove this :not([disabled]) by adding a pointer-events set to none when disabled ?